lettuce/io.py

```diff
@@ -48,9 +48,14 @@ def __call__(self, i, t, f):
             t = self.flow.units.convert_time_to_pu(t)
             u = self.flow.units.convert_velocity_to_pu(self.lattice.u(f))
             p = self.flow.units.convert_density_lu_to_pressure_pu(self.lattice.rho(f))
-            self.point_dict["p"] = self.lattice.convert_to_numpy(p[0, ..., None])
-            for d in range(self.lattice.D):
-                self.point_dict[f"u{'xyz'[d]}"] = self.lattice.convert_to_numpy(u[d, ..., None])
+            if self.lattice.D == 2:
+                self.point_dict["p"] = self.lattice.convert_to_numpy(p[0, ..., None])
+                for d in range(self.lattice.D):
+                    self.point_dict[f"u{'xyz'[d]}"] = self.lattice.convert_to_numpy(u[d, ..., None])
+            else:
+                self.point_dict["p"] = self.lattice.convert_to_numpy(p[0, ...])
+                for d in range(self.lattice.D):
+                    self.point_dict[f"u{'xyz'[d]}"] = self.lattice.convert_to_numpy(u[d, ...])
```

Comment:
Can you remind me: Why do we need 2 separate cases here? Does pyevtk require a different shape for 2D and 3D?

Reply:
Yes it does. See https://github.com/Olllom/lettuce/pull/14